pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java

```diff
@@ -165,6 +168,11 @@ public void init() throws DBException {
         if (startError) {
           /* or create it, if it doesn't exist */
           try {
+            if (size == null) {
```

Comment:
so, the size cannot be specified in the json config?

Reply:
as I specified in the readme - I'd use path and size as a dedicated params and json file just for additional, extra params (like for radix). I believe it's easier for potential users just to specify these 2 params from the command line, then editing config file.
But... I guess we can make this work and throw errors from pmemkv if anything goes sideways.